### ISSUE TITLE
CR Snapshot Update Request for Web Neural Network API - webnn

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/transitions#592]
# Document URLs
https://webmachinelearning.github.io/webnn/

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/public-webmachinelearning-wg/2024Mar/0000.html

# Link to previous Candidate Recommendation transition or update request
https://github.com/w3c/transitions/issues/491

# Substantive changes
Since the [initial Candidate Recommendation Snapshot](https://www.w3.org/TR/2023/CR-webnn-20230330/), the Working Group has gathered further [implementation experience](https://webmachinelearning.github.io/webnn-status/) and added new operations and data types needed for well-known transformers https://github.com/webmachinelearning/webnn/issues/375. In addition, the group has removed selected features informed by this implementation experience: higher-level operations that can be expressed in terms of lower-level primitives in a performant manner, and support for synchronous execution. The group has also updated the specification to use modern authoring conventions to improve interoperability and precision of normative definitions and is developing a new feature, a https://github.com/webmachinelearning/webnn/issues/482, to improve performance and interoperability between the WebNN, WebGPU APIs and purpose-built hardware for ML.

# Any changes in requirements?
None

# Wide Review of substantive changes
https://github.com/webmachinelearning/webnn/issues/239 (updated TAG review requested a month ago, but still untriaged)

# Issues status
There are [49 substantive issues](https://github.com/webmachinelearning/webnn/issues?q=is%3Aissue+is%3Aopen+-label%3Aquestion+-label%3A%22feature+request%22++-label%3Aeditorial+-label%3Aconventions+-label%3Atesting++-label%3A%22use+case%22++-label%3Aprocess), mostly related to integration with WebGPU, and improvements to the definitions of existing operations.

# Formal Objections
None

# Any changes in implementation information?
Active implementation work in Chromium, shipping behind a flag in Chrome and Edge
https://webmachinelearning.github.io/webnn-status/

# Deadline for further comments
publication date + 6 months

# Any changes in patent disclosures?
None

See [Requirements for updating a Candidate Recommendation Snapshot](https://www.w3.org/Guide/transitions?profile=CR&cr=snapshot) for further information.

cc @anssiko

**Comment by plehegar** [comment]
(waiting on TAG session this Thursday)

**Comment by anssiko** [comment]
@plehegar, PING was informed in https://github.com/w3cping/privacy-request/issues/96#issuecomment-1911928615

**Comment by plehegar** [comment]
@anssiko , without opening a new issue, this was missed by PING. I'll check with them.

**Comment by plehegar** [comment]
Please, in the future, open a new issue when you request a new review from horizontal groups.

**Comment by plehegar** [comment]
Approved

**Comment by xueyuanjia** [comment]
draft transition received.
